From d0c1f075ae4073e81f2b75fa4f45e432a495afb7 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Thu, 14 Jul 2022 16:14:37 +0530
Subject: [PATCH] meson_options: Bump max value of platform-sdk-version to 33

During building Android-13, the following error appears:
meson.build:21:0: ERROR: New value 33 is more than maximum value 31.

Signed-off-by: svenate <salini.venate@intel.com>
---
 meson_options.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson_options.txt b/meson_options.txt
index 1f6ef385561..da9e6443e8f 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -455,7 +455,7 @@ option(
   'platform-sdk-version',
   type : 'integer',
   min : 25,
-  max : 31,
+  max : 33,
   value : 25,
   description : 'Android Platform SDK version. Default: Nougat version.'
 )
-- 
2.17.1

